lib/theme.ts

```diff
@@ -10,7 +9,7 @@ declare module '@material-ui/core/styles/createTypography' {
     }
 }
 
-const THEME: ThemeOptions = {
+const THEME: DeprecatedThemeOptions = {
```

Comment:
Curious how different the new Theme options are. If we use the new `ThemeOptions` type, are there lots of conflicts?